src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -104,7 +104,7 @@ public override IntermediateNode Visit(TexlTableNode node, IRTranslatorContext c
                 var children = node.Children.Select(child => child.Accept(this, context)).ToArray();
                 var irContext = context.GetIRContext(node);
 
-                if (!context.Binding.Features.HasTableSyntaxDoesntWrapRecords() || (children.Any() && children.First() is not RecordNode))
+                if (!context.Binding.CheckTypesContext.Features.HasTableSyntaxDoesntWrapRecords() || (children.Any() && children.First() is not RecordNode))
```

Comment:
>CheckTypesContext [](http://example.com/codeflow?start=37&length=17)

Looks like Features is still on binding, is this needed?

Reply:
I left features on Binding to prevent breaking too much in PAClient, but it will be removed over time.